src/AccountCreated.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import Button  from 'react-bootstrap/Button';
```

Comment:
I had it before but I tried to run it in React it butchers my code. Is there a special way it works on React?

Reply:
What do you mean by "butchered"? It will automatically fix all the wrong identations, be consistent with `"`/`', fix too long lines, etc.

If it's the too long lines, you add add the configuration file I mentioned and set `"printWidth": 100` in that file to increase the limit :)